recipes/thorvg/all/conanfile.py

```diff
@@ -131,7 +137,8 @@ def generate(self):
             "threads": bool(self.options.with_threads),
             "examples": bool(self.options.with_examples),
             "tests": False,
-            "log": is_debug
+            "log": is_debug,
+            "static": True,
```

Comment:
As far as I saw, this option is used [here](https://github.com/search?q=repo%3Athorvg%2Fthorvg%20get_option(%27static%27)&type=code). It would be great to clarify why this is always `true` because the upstream option description sounds a bit confusing.

Reply:
Thanks a lot for your comment!
I'm sorry it is my debug code.
Removed.